src/components/ReportActionItem/TaskPreview.js

```diff
@@ -65,7 +66,7 @@ function TaskPreview(props) {
         : props.action.childStateNum === CONST.REPORT.STATE_NUM.SUBMITTED && props.action.childStatusNum === CONST.REPORT.STATUS.APPROVED;
     const taskTitle = props.taskReport.reportName || props.action.childReportName;
     const taskAssigneeAccountID = TaskUtils.getTaskAssigneeAccountID(props.taskReport);
-    const taskAssignee = lodashGet(props.personalDetailsList, [taskAssigneeAccountID, 'login'], '');
+    const taskAssignee = lodashGet(props.personalDetailsList, [taskAssigneeAccountID, 'login'], lodashGet(props.personalDetailsList, [taskAssigneeAccountID, 'displayName'], ''));
```

Comment:
personally not crazy about this nested lodash, would rather see something like:
```suggestion
    const assigneeLogin = lodashGet(props.personalDetailsList, [taskAssigneeAccountID, 'login'], '');
    const assigneeDisplayName = lodashGet(props.personalDetailsList, [taskAssigneeAccountID, 'displayName'], '');
    const taskAssignee = assigneeLogin || assigneeDisplayName;
```

as it's more clear to me that `displayName` is a fall back if we don't have a login. instead of trying to reason how the 2nd lodashGet might get called and be a fallback for the first call.

Reply:
Ooh that's totally fair - I'll try to remember to clean this up later - i can see it being useful but not highest priority at the moment

Reply:
Cleanup PR here: https://github.com/Expensify/App/pull/22439